tools/cli/workflow.go

```diff
@@ -143,6 +143,11 @@ func newWorkflowCommands() []cli.Command {
 					Usage: "Optional input for the workflow from JSON file. If there are multiple JSON, concatenate them and separate by space or newline. " +
 						"Input from file will be overwrite by input from command line",
 				},
+				cli.StringFlag{
+					Name: FlagWorkflowIdReusePolicyAlias,
+					Usage: "Optional input to configure if the same workflow ID is allow to use for new workflow execution (choose from 0 to 2). " +
```

Comment:
Could we explicit present all options like `0: AllowDuplicateFailedOnly, 1: AllowDuplicate, 2: RejectDuplicate` to save user another link click?

Reply:
Done.